File: projects/project 1/source/myPA1/char_counter/char_counter/MainActivity.cs
using Android.App;
using Android.Widget;
using Android.OS;

namespace char_counter
{
    [Activity(Label = "Character Counter", MainLauncher = true, Icon = "@drawable/icon")]
    public class MainActivity : Activity
    {

        protected override void OnCreate(Bundle savedInstanceState)
        {
            base.OnCreate(savedInstanceState);

            // Set our view from the "main" layout resource
            SetContentView(Resource.Layout.Main);



            string len, myresult;
            Button clear = FindViewById<Button>(Resource.Id.ClearButton);

            EditText localtext = FindViewById<EditText>(Resource.Id.toBeCounted);



            localtext.TextChanged += delegate
            {
                len = FindViewById<EditText>(Resource.Id.toBeCounted).Text.ToString();
                myresult = FindViewById<TextView>(Resource.Id.printnum).ToString();
                len = len.Length.ToString();

                myresult = len + " characters long";

                FindViewById<TextView>(Resource.Id.printnum).Text = myresult;
            };

            clear.Click += delegate
            {
                FindViewById<EditText>(Resource.Id.toBeCounted).Text = "";
                FindViewById<TextView>(Resource.Id.printnum).Text = "0 characters long";
            };
        }
    }
}

File: projects/project 1/fullproject1/Hangman/project1use/app/src/main/java/com/wmd42/hangman/MainActivity.java
package com.wmd42.hangman;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;
import java.util.Arrays;

public class MainActivity extends AppCompatActivity {

    private boolean isclicked = false; //to see if the button has been clicked yet
    private int chances = 6;
    private String ourword;   //the word inputted buy the user to be guessed
    private char[] wordchars; //contains the word to be guessed
    private char[] strguess; //contains current guesses and blank spots
    private String guessdisplay; //to be passed to TextView to display current guesses
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void clicked(View view){

        //if this is the first click, initialize the word guess and strguess arrays
        if(!isclicked){
            TextView numgues = (TextView) findViewById(R.id.chancesLeft);
            TextView letgues = (TextView) findViewById(R.id.guessedLetter);
            EditText inbox = (EditText) findViewById(R.id.Input);
            ourword = inbox.getText().toString();
            wordchars = new char[ourword.length()];
            wordchars = ourword.toCharArray();
            strguess = new char[ourword.length()];
            Log.i("test",ourword);
            //fill guess array with underscores
            for(int i = 0; i < ourword.length(); i++){
                strguess[i] = '_';
                Log.i("guessarray", Character.toString(strguess[i]));
            }

            //display guesses

            CharSequence guessdisplay = Arrays.toString(strguess);
            Log.i("underscores", (String)guessdisplay);
            letgues.setText(guessdisplay);

            //displays chances left
            numgues.setText((CharSequence)String.format("You have %d chances left", chances));

            isclicked = true;

        }
    }

}
